plugins/aws-appmesh/plugin/commands.go

```diff
@@ -359,6 +360,13 @@ func forEachSlice(inputPorts []string, maximumPort int, run func([]string) error
 			log.Errorf("multiport limit: %d exceeded, ignoring remaining rules", maximumPort)
 			break
 		}
+
+		//There should not be empty string in port list here
```

Comment:
If this is a real possibility that should be validated against, then do the validation in validateConfig. These input problems should be caught in the beginning of execution, not all the way when we insert iptables rules.

Reply:
Deleted this